### PR TITLE
fix: zsh script

### LINKED ISSUE #2309: [bug]: zsh script doesn't work for vscode-extension
### 👀 Is there an existing issue for this?

- [X] I have searched and didn't find similar issue

### 👍 Current behavior

Currently vscode extension runs all the command in bash by default even if the default shell for the system is zsh.

### 👟 Steps to Replicate

1. Go to Generate Integration Tests
2. Run Record , it will run in bash 

### 🧱 Your Environment

_No response_

### 💻 Operating system

Windows

### 🎲 Version

Latest Version

### 💻 Repository

vscode-extension

### PR DESCRIPTION
**Pull request by ayush3160**
Fixed zsh script and changed the one click install command.

fixes [#2309](https://github.com/keploy/keploy/issues/2309)

### CODE REVIEW

File: scripts/zsh/keploy_record_script.zsh [hunk 1]
```diff
@@ -1,53 +1,85 @@
...
+  sudo -E env "PATH=$PATH" keploy record  > "$fifo" 2>&1
+else
+  keploycmd="sudo -E env PATH=\"$PATH\" keploy"
+  $keploycmd record > "$fifo" 2>&1
+fi
 
 # Clean up: Wait for keploy command to finish
 wait $!
+touch ./log_file.txt
+
```
Comment (shivamsouravjha) [inline scripts/zsh/keploy_record_script.zsh]: why this???

File: scripts/zsh/keploy_record_script.zsh [hunk 2]
```diff
@@ -1,53 +1,85 @@
...
-
 if [[ "$command" =~ .*"go".* ]]; then
-  # echo "Go is present."
   go mod download
   go build -o application
-fi 
 
-# Adding sudo here worked
-keploycmd="sudo -E env PATH=\"$PATH\" keploy"
+elif [[ "$command" =~ .*"python3".* ]]; then
```
Comment (shivamsouravjha) [inline scripts/zsh/keploy_record_script.zsh]: wrong removal

File: scripts/zsh/keploy_record_script.zsh [hunk 3]
```diff
@@ -1,53 +1,85 @@
...
+keploy_config="keploy.yml"
+if [[ ! -f "$keploy_config" ]]; then
+    echo "keploy.yml file not found in the current directory."
+    exit 1
+fi
+
+command=$(awk '/command:/ { $1=""; sub(/^ /, ""); print }' "$keploy_config")
+# echo "Command in yml file: $command"
+
+# Check if command is empty
```
Comment (shivamsouravjha) [inline scripts/zsh/keploy_record_script.zsh]: lets remove these comments

**Review by khareyash05 — APPROVED** [review]
LGTM! Not able to replicate on Bash shell but works on @ayush3160 when he sets his default terminal to bash


Also have to check that in test sudo is 2 times required

**Review by khareyash05 — APPROVED** [review]
LGTM! But above bash comment remains still

**Pull request merged**